topics/dev/tutorials/tool-from-scratch/tutorial.md

```diff
@@ -399,6 +399,29 @@ user interface, then translate the user's selections back into the command to be
 on writing a wrapper. Although this section will cover the full process and many of the options available to
 anyone wrapping a tool, a more complete list is available in [the Galaxy tool docs](https://docs.galaxyproject.org/en/latest/dev/schema.html).
 
+## Galaxy Language Server
+
+Although Galaxy tools can be written by hand using the resources below and the [Galaxy documentation](https://docs.galaxyproject.org/en/latest/dev/schema.html) as needed, the [Galaxy Language Server](https://github.com/galaxyproject/galaxy-language-server) for VSCode has been developed to ease and speed the wrapping of new tools. The plugin contains the following features that will benefit any developer working in Galaxy tools:
+
+- Tag and attribute auto-completion
+- Documentation on Hover
+- Document validation
+- Document auto-formatting
+- Tag auto-closing
+- Snippets
+- Embedded syntax highlighting
+- Auto-generate tests
+- Auto-generate command section
+- Auto-sort param attributes
+- Run planemo tests in the Test Explorer
+- Improved macros support
+- Extract macro
+- Full Galaxy tool linter integration
+
+To install, go to the `extensions` tab in VSCode, search "Galaxy Tools", and click install. Make sure that the python version available in your environment is compatible, and install updates as necessary, as requested by the alerts in VSCode. Once installed, simply make sure the document format is `Galaxy Tool Wrapper` when working on a tool to ensure the laungage server is enabled.
```

Comment:
Could you please add a link, like in the github repo: https://github.com/davelopez/galaxy-workflows-vscode/#galaxy-workflows-vscode-extension "click to install in vscode", that would be a much nicer user experience than "search for this string".